client/app/domain/email-obfuscation/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+  <translation id="domain_email_obfuscation_back_button">Régénérer les adresses e-mail de substitution</translation>
+  <translation id="domain_email_obfuscation_label">Sélectionner les contacts pour lesquels vous souhaitez régénérer les adresses e-mail de substitution :</translation>
```

Comment:
```suggestion
  <translation id="domain_email_obfuscation_label">Sélectionner les contacts pour lesquels vous souhaitez régénérer les adresses e-mail de substitution :</translation>
```
:)

Reply:
@FredericEspiau I can't see the diff with your suggestion

Reply:
Insecable space 😄